hibernate-community-dialects/src/main/java/org/hibernate/community/dialect/InformixDialect.java

```diff
@@ -177,7 +200,7 @@ protected void registerColumnTypes(TypeContributions typeContributions, ServiceR
 		//double precision.
 		ddlTypeRegistry.addDescriptor(
 				CapacityDependentDdlType.builder( FLOAT, "float($p)", this )
```

Comment:
```suggestion
				CapacityDependentDdlType.builder( FLOAT, "float", this )
```

@beikov if you're suggesting to just make this one-line change and leave the rest of this patch as it is, then I agree.

Reply:
Yeah, I think this change should be made. I also think the other changes then become obsolete, but if you insist on keeping that, I'm ok with that.

Reply:
Yeah, my reading of the Informix docs is that the other changes make sense.

Reply:
I tested the proposed change of using float without precision. In this case, it is not necessary to have a custom SizeStrategy implementation.